go.mod

```diff
@@ -6,10 +6,10 @@ require (
 	github.com/Azure/azure-sdk-for-go v61.4.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.24
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
-	github.com/IBM-Cloud/power-go-client v1.1.4
-	github.com/IBM/go-sdk-core/v5 v5.9.2
+	github.com/IBM-Cloud/power-go-client v1.1.9
+	github.com/IBM/go-sdk-core/v5 v5.10.1
 	github.com/IBM/networking-go-sdk v0.29.0
-	github.com/IBM/platform-services-go-sdk v0.22.8
+	github.com/IBM/platform-services-go-sdk v0.24.1
 	github.com/IBM/vpc-go-sdk v1.0.1
```

Comment:
wondering if you want to change this package as well?

Reply:
Updated.